src/job/jobHandler.ts

```diff
@@ -635,27 +655,43 @@ export abstract class JobHandler {
     return this._logger;
   }
 
-  protected async previewWebhook(): Promise<AxiosResponse<string>> {
-    const previewWebhookURL = 'https://webhook.gatsbyjs.com/hooks/data_source';
-    const githubUsername = this.currJob.user;
-    const gatsbySiteId = await this._repoEntitlementsRepo.getGatsbySiteIdByGithubUsername(githubUsername);
-    if (!gatsbySiteId) {
-      const message = `User ${githubUsername} does not have a Gatsby Cloud Site ID.`;
-      this._logger.warn('Gatsby Cloud Preview Webhook', message);
-      throw new Error(message);
-    }
+  // Invokes Gatsby Cloud Preview Webhook
+  protected async callGatsbyCloudWebhook(): Promise<void> {
+    const featurePreviewWebhookEnabled = process.env.GATSBY_CLOUD_PREVIEW_WEBHOOK_ENABLED;
+    // Logging for Debugging purposes only will remove once we see the build working in Gatsby.
+    await this.logger.save(
+      this.currJob._id,
+      `${'(GATSBY_CLOUD_PREVIEW_WEBHOOK_ENABLED)'.padEnd(15)}${featurePreviewWebhookEnabled}`
+    );
 
-    const url = `${previewWebhookURL}/${gatsbySiteId}`;
-    return await axios.post(
-      url,
-      {
-        jobId: this.currJob._id,
-        status: this.currJob.status,
-      },
-      {
-        headers: { 'x-gatsby-cloud-data-source': 'gatsby-source-snooty-preview' },
+    try {
+      const previewWebhookURL = 'https://webhook.gatsbyjs.com/hooks/data_source';
+      const githubUsername = this.currJob.user;
+      const gatsbySiteId = await this._repoEntitlementsRepo.getGatsbySiteIdByGithubUsername(githubUsername);
+      if (!gatsbySiteId) {
+        const message = `User ${githubUsername} does not have a Gatsby Cloud Site ID.`;
+        this._logger.warn('Gatsby Cloud Preview Webhook', message);
+        throw new Error(message);
       }
-    );
+
+      const url = `${previewWebhookURL}/${gatsbySiteId}`;
+      const response = await axios.post(
+        url,
+        {
+          jobId: this.currJob._id,
```

Comment:
I noticed that the the post request had the `status` property as a part of the object body. Was this intentionally removed?

Reply:
Yes, this is intentional. Right now, the source plugin doesn't do anything with the `status` (which should always be `inProgress` in this case) that it receives